js/code.js

```diff
@@ -310,12 +343,12 @@ function searchContacts()
 						phoneElement.innerHTML = phoneNumber.substring(0, 3) + "-" + phoneNumber.substring(3, 6) + "-" + phoneNumber.substring(6, 10);
 
 						let updateElement = contactElement.appendChild(document.createElement("td"));
-						updateElement.setAttribute("id", "update");
+						updateElement.setAttribute("id", jsonObject.results[i].UID);
```

Comment:
Made the UPDATE button id change you suggested. It worked out great! @CathyChian